src/node_modules/editor/index.js

```diff
@@ -32,7 +32,22 @@ const output = CodeMirror.fromTextArea(
   document.getElementById('output'),
   Object.assign({}, opts, { readOnly: true })
 )
-
+async function clearDocumentWindow() {
+  await sandbox.webContents.executeJavaScript(`
+    try {
+      eval(\`(function() { document.body.innerHTML='' })()\`)
```

Comment:
I was thinking maybe we also would want to clear the javascript state in the window. Would it be easier for us though to just recreate the document window?

Reply:
Ya, that would be more easier.They dont expose any methods to do so.While i was reading [Node VM](https://nodejs.org/api/vm.html) came across.Here we do have an option of clearing the context.

Reply:
@0x00A Do advice how to proceed with this.

Reply:
Actually, the original version of this used node's VM module, but the problem with that is it handles uncaught exceptions poorly, which is exactly what we need in this use case.

Reply:
@0x00A I am a bit confused.Are you recommending to change it to node vm??:D